reference/policy.md

```diff
@@ -4,51 +4,51 @@
 
 The Cloud Native Maturity Model is composed of five separate documents - the [Prologue](./prologue.md) and the four key reference documents:
 
-* [People](./people.md)  
-* [Process](./process.md)  
-* [Policy](./policy.md)  
+* [People](./people.md)
+* [Process](./process.md)
+* [Policy](./policy.md)
 * [Technology](./technology.md)
 
 ## Introduction
 
-The Cloud Native Maturity Model covers four major dimensions - People, Process, Policy and Technology. This paper addresses policy. Your goal of cloud native should be to implement policy to embrace shift left security. Across all levels, you will seek to implement policies and automate enforcement in CI/CD and continuously monitor against policies in production.
+The Cloud Native Maturity Model covers four major dimensions - People, Process, Policy and Technology. This paper addresses policy, the critical implementation of internal requirements, and compliance to external regulations. Your goal as a cloud native practitioner should be to implement policies that enable and encourage shift left security. As your maturity grows, you will seek to automate policy enforcement in CI/CD and continuously monitor against policies in production.
```

Comment:
"shift left" with or without capitals? I'm confused by even the CNCF Glossary using both forms with no apparent reasons.